Bottom: 2377b6f81c8b0fcdc3e69870fb835e89121b808e
Top:    59ba5b568f4f8946b9ce22d6aec1e983695e1368
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 15:57:53 +0100

Refresh of autoroute-c-simplify-some-long

---

diff --git a/src/autoroute.c b/src/autoroute.c
index 554fda7..1669a36 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -4790,13 +4790,7 @@ RouteAll (routedata_t * rd)
 	  if (ras.total_subnets == 0)
 #endif
 	    {
-	      /* Route easiest nets from this pass first on next pass.
-	       * This works best because it's likely that the hardest
-	       * is the last one routed (since it has the most obstacles)
-	       * but it will do no good to rip it up and try it again
-	       * without first changing any of the other routes
-	       */
-	      heap_insert (next_pass, total_net_cost, net);
+	      heap_insert (next_pass, 0, net);
 	      continue;
 	    }
